cpp/Agents/EvolutionStrategy.cpp

```diff
@@ -0,0 +1,13 @@
+#include "EvolutionStrategy.hpp"
+
+int main(int argc, char* argv[]){
```

Comment:
Here we can read the variable values from arguments, I didn't know library for that so I skipped for now